p4-16/psa/PSA.mdk

```diff
@@ -1497,6 +1499,59 @@ table.
 A PSA implementation must implement tables that have both a
 `psa_direct_counter` and `psa_direct_meter` property.
 
+### Table entry timeout notification {#sec-idle-timeout}
+
+PSA uses the `psa_idle_timeout` to enable a table implementation send
+notifications from the PSA device when a configurable time has passed since an
+entry was last matched. The property may take one of two values --
+`PSA_NO_TIMEOUT`, and `PSA_NOTIFY_CONTROL`. `PSA_NO_TIMEOUT` disables idle
```

Comment:
maybe you should say that `PSA_NO_TIMEOUT` is the default when the property is omitted